apps/app/src/client/util/apiv3-client.ts

```diff
@@ -50,7 +50,7 @@ export async function apiv3Post<T = any>(path: string, params: unknown = {}): Pr
 }
 
 export async function apiv3PostForm<T = any>(path: string, formData: FormData): Promise<AxiosResponse<T>> {
-  return apiv3Post<T>(path, formData);
+  return apiv3Request('postForm', path, formData);
```

Comment:
ref) https://github.com/bmuenzenmeyer/axios-1.0.0-migration-guide/?tab=readme-ov-file#multipart-form-data-is-no-longer-automatically-set